### DISCUSSION TITLE
Datadog dashboard

### DISCUSSION THREAD

**Discussion by invalidred** [cloudnative-pg/cloudnative-pg#3307] [Show and tell]
Hey team.

At my company we use datadog for observability and I was not able to find a dashboard that worked out of box with the default metrics provided CNPG.

Here is a default dashboard that should work with default metrics provided by CNPG (with minor tweaks to template variables).

https://github.com/invalidred/cnpg-datadog/blob/main/cnpg-dashboard.json

**Comment by invalidred** [comment]
Here are some images from that dashbaord

<img width="1230" alt="Screenshot 2023-11-06 at 4 48 23 PM" src="https://github.com/cloudnative-pg/cloudnative-pg/assets/5270458/180821ac-6402-460e-8957-33a86f194bc0">

<img width="1213" alt="Screenshot 2023-11-06 at 4 48 35 PM" src="https://github.com/cloudnative-pg/cloudnative-pg/assets/5270458/26186fa0-d9bd-472e-ae1c-bc93b3d6980e">

<img width="1237" alt="Screenshot 2023-11-06 at 4 48 45 PM" src="https://github.com/cloudnative-pg/cloudnative-pg/assets/5270458/655545a6-a604-41c5-adfa-8c5b62684dd8">

**Comment by gbartolini** [comment]
Out of curiosity, would you be interested in officially maintaining this in a separate repository under the cloudnativepg organizations?

**Reply by invalidred** [reply]
That would be wonderful. More than happy to help as needed @gbartolini.

**Reply by panteparak** [reply]
@gbartolini @invalidred 

out of curiosity where is this repo being kept?
